flybywiresim/a32nx
# [Lights] DIM/BRT Lights | Overhead Fixed

## Summary of Changes
**Interior Lights**

- I have re-worked the interior lights adding a more realistic look. You can now see the overhead as well based on which option you have set it to. (Dim or Bright)

# Screenshots
### **DIM Lights**
![Microsoft Flight Simulator 10_11_2020 2_52_52 PM](https://user-images.githubusercontent.com/43104362/95678297-60cdbb80-0bd4-11eb-8125-013a58297e91.png)
![Microsoft Flight Simulator 10_11_2020 2_54_28 PM (1)](https://user-images.githubusercontent.com/43104362/95678295-5f03f800-0bd4-11eb-911a-1e0d45efbc58.png)
![Microsoft Flight Simulator 10_11_2020 2_55_48 PM](https://user-images.githubusercontent.com/43104362/95678292-5ca19e00-0bd4-11eb-823f-00c5ffc149b1.png)

### **BRT Lights**
![Microsoft Flight Simulator 10_11_2020 2_56_06 PM](https://user-images.githubusercontent.com/43104362/95678325-9ffc0c80-0bd4-11eb-90a2-bd9345d9a98b.png)
![Microsoft Flight Simulator 10_11_2020 2_54_45 PM](https://user-images.githubusercontent.com/43104362/95678326-a25e6680-0bd4-11eb-9db0-432a3e1cac84.png)
![Microsoft Flight Simulator 10_11_2020 2_53_02 PM](https://user-images.githubusercontent.com/43104362/95678353-dc2f6d00-0bd4-11eb-907a-e3ae7fc979bf.png)

# BEFORE
![Screenshot_82](https://user-images.githubusercontent.com/43104362/95678405-1b5dbe00-0bd5-11eb-8736-cd7d1b1a712d.png)

## References

Cockpit Screenshots showing the DIM/BRT are lacking. However, I managed to ask a few IRL Pilots to approve my work.
[FSLabs Reference](https://youtu.be/HZMrkJFLeso?t=117)

![image](https://user-images.githubusercontent.com/43104362/95678606-662c0580-0bd6-11eb-96bf-3ce4315f244e.png)
![image](https://user-images.githubusercontent.com/43104362/95678626-7cd25c80-0bd6-11eb-8b19-141bded440e1.png)





## Additional Context

Discord Username: `Harry ✈#0001`

Please let me know your feedback/opinions so I can make changes to this PR.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The `build.py` will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
2. Click on the **Checks** tab on the PR
3. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link
